pennylane/resource/resource.py

```diff
@@ -71,6 +75,35 @@ def _ipython_display_(self):
         print(str(self))
 
 
+class OperationWithResources(Operation):
```

Comment:
Will `OperationWithResources` be top-level accessible? `qml.OperationWithResources`?

Reply:
It currently isn't, I can make it top level accessible